src/apps/chifra/pkg/cache/paths.go

```diff
@@ -0,0 +1,49 @@
+package cache
+
+import (
+	"fmt"
+	"path"
+	"strings"
+
+	"github.com/TrueBlocks/trueblocks-core/src/apps/chifra/pkg/types"
+)
+
+var itemToDirectory = map[CacheItem]string{
```

Comment:
I wrote code similar to this recently, but I like your code better. I anticipated this, so I have no problem removing my code and using this. We can do that after we merge. There's a slightly different way of figuring out `path` for `monitors` and `reconciliations`, but they will easily fit into this structure. Looks good.